docs/rules/terraform_dash_in_resource_name.md

```diff
@@ -0,0 +1,26 @@
+# terraform_dash_in_resource_name
+
+Disallow dashes (-) in `resource` names without.
+
+## Example
+
+```hcl
+resource "aws_eip" "dash-name" {
+}
+
+resource "aws_eip" "no_dash_name" {
+}```
```

Comment:
Looks like this line is breaking the markdown...
https://github.com/wata727/tflint/blob/3d63487837042640096def4b234b5b0832af2d14/docs/rules/terraform_dash_in_resource_name.md

Reply:
Fixed!